src/Concerns/HasBotsAndChats.php

```diff
@@ -56,6 +56,11 @@ public function chat(TelegraphChat|string $chat): Telegraph
         return $telegraph;
     }
 
+    public function thread(int $thread): Telegraph
```

Comment:
@ahr1384 maybe this could be a more expressive API as `->onThread(int $thread)`, what do you think?